src/java/org/apache/cassandra/service/StorageService.java

```diff
@@ -6180,17 +6245,42 @@ public void setBatchSizeFailureThreshold(int threshold)
         logger.info("updated batch_size_fail_threshold to {}", threshold);
     }
 
+    @Deprecated
+    @Override
     public int getBatchSizeWarnThreshold()
     {
         return DatabaseDescriptor.getBatchSizeWarnThresholdInKiB();
     }
 
+    @Deprecated
+    @Override
     public void setBatchSizeWarnThreshold(int threshold)
     {
         DatabaseDescriptor.setBatchSizeWarnThresholdInKiB(threshold);
         logger.info("Updated batch_size_warn_threshold to {}", threshold);
     }
 
+    @Override
+    public int getBatchSizeWarnThresholdInKiB()
```

Comment:
same comment as above, can we document "why" the new method that has identical logic?